ewcfg.py

```diff
@@ -3193,6 +3194,31 @@ def wef_umbrella(ctn = None):
 	elif aim >= (10 - int(10 * ctn.crit_mod)):
 		ctn.crit = True
 		ctn.slimes_damage *= 2
+# weapon effect function for "minecraft bow"
+def wef_bow(ctn = None):
+	aim = (random.randrange(0, 13) - 2)
+	user_mutations = ctn.user_data.get_mutations()
+	dmg = ctn.slimes_damage
+	ctn.sap_damage = 1
+	ctn.sap_ignored = 8
+	
+	# Increased miss chance if attacking within less than two seconds after last attack
+	time_lastattack = ctn.time_now - (float(ctn.weapon_item.item_props.get("time_lastattack")) if ctn.weapon_item.item_props.get("time_lastattack") != None else ctn.time_now)
+	ctn.miss_mod += (((10 - min(time_lastattack, 10)) / 10) ** 2) / 13 * 10
+
+	ctn.slimes_damage = int(ctn.slimes_damage * 4)
+
+	if aim <= (-2 + int(16 * ctn.miss_mod)):
```

Comment:
why did you change this one to 16 instead of the other one to 13?

Reply:
Uh, is it bad if  i do that?